macros/LearnSidebar.ejs

```diff
@@ -187,6 +187,34 @@ var text = mdn.localStringMap({
         'Package_management_basics' : 'Package management basics',
         'Introducing_a_complete_toolchain' : 'Introducing a complete toolchain',
         'Deploying_our_app' : 'Deploying our app',
+      'Introduction_to_client-side_frameworks': 'Introduction to client-side frameworks',
+        'Client-side_frameworks_overview': 'Client-side frameworks overview',
+        'Framework_main_features': 'Framework main features',
+      'React': 'React',
+        'Getting_started_with_React': 'Getting started with React',
+        'Beginning_our_React_todo_list': 'Beginning our React todo list',
+        'Componentizing_our_React_app': 'Componentizing our React app',
+        'React_interactivity:_Events_and_state': 'React interactivity: Events and state',
+        'React_interactivity:_Editing_filtering_conditional_rendering': 'React interactivity: Editing, filtering, conditional rendering',
+        'Accessibility_in_React': 'Accessibility in React',
+        'React_resources': 'React resources',
+      'Ember': 'Ember',
+        'Getting_started_with_Ember': 'Getting started with Ember',
+        'Ember_app_structure_and_componentization': 'Ember app structure and componentization',
+        'Ember_interactivity:_Events_classes_and_state': 'Ember interactivity: Events, classes and state',
+        'Ember_Interactivity:_Footer_functionality_conditional_rendering': 'Ember Interactivity: Footer functionality, conditional rendering',
+        'Routing_in_Ember': 'Routing in Ember',
+        'Ember_resources_and_troubleshooting': 'Ember resources and troubleshooting',
+      'Vue': 'Vue',
+        'Getting_started_with_Vue': 'Getting started with Vue',
+        'Creating_our_first_Vue_component': 'Creating our first Vue component',
+        'Rendering_a_list_of_Vue_components': 'Rendering a list of Vue components',
+        'Adding_a_new_todo_form:_Vue_events methods and models': 'Adding a new todo form: Vue events, methods, and models',
```

Comment:
Some underscores missing here and this needs fixing throughout. The text doesn't render right now.

```suggestion
        'Adding_a_new_todo_form:_Vue_events_methods_and_models': 'Adding a new todo form: Vue events, methods, and models',
```